### PR TITLE
New page: CSS `d` property

### PR DESCRIPTION
**Pull request by estelle**
part of https://github.com/mdn/content/issues/34763

### CODE REVIEW

File: files/en-us/web/svg/attribute/d/index.md [hunk 1]
```diff
@@ -103,6 +103,8 @@ svg {
 
 {{EmbedLiveSample('Using d as a CSS Property', '100%', 200)}}
 
+For an =`<path>` animation example, see the CSS {{cssxref("d")}} property reference page example.
```
Comment (Josh-Cena) [inline files/en-us/web/svg/attribute/d/index.md:106]:
```suggestion
For a `<path>` animation example, see the CSS {{cssxref("d")}} property reference page example.
```

File: files/en-us/web/css/d/index.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+```css
+/* Default */
+d: none;
+
+/* Basic usage */
+d: path("m 5,5 h 35 L 20,30 z")
+d: path("M 0,25 l 50,150 l 200,50 z")
+d: path("M 10,5 l 90,0 -80,80 0,-60 80,80 -90,0 z")
+
+/* with
```
Comment (Josh-Cena) [inline files/en-us/web/css/d/index.md:26]: Remove

Comment (estelle) [inline files/en-us/web/css/d/index.md:26]:
```suggestion
```

**Review by estelle — COMMENTED** [review]
Thanks @Josh-Cena

File: files/en-us/web/css/d/index.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+d: inherit;
+d: initial;
+d: revert;
+d: revert-layer;
+d: unset;
+```
+
+### Values
+
+The value is either a {{cssxref("path")}} function with a single {{cssxref("&lt;string&gt;")}} parameter or the keyword `none`.
```
Comment (Josh-Cena) [inline files/en-us/web/css/d/index.md:36]:
```suggestion
The value is either a {{cssxref("basic-shape/path", "path()")}} function with a single {{cssxref("&lt;string&gt;")}} parameter or the keyword `none`.
```

File: files/en-us/web/css/d/index.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+```html
+<svg>
+  <path />
+</svg>
+```
+
+#### CSS
+
+We use the `d` attribute to define a heart with a line through it. We use CSS to define the {{cssxref("fill")}}, {{cssxref("stroke")}}, and {{cssxref("stroke-width")}} of that path, and add a two-second {{cssxref("transition")}}. We add a {{cssxref(":hover")}} style that contains a slightly different {{cssxref("path()")}} function; the path has the same number of data points as the default state, making the path animatable.
```
Comment (Josh-Cena) [inline files/en-us/web/css/d/index.md:111]:
```suggestion
We use the `d` attribute to define a heart with a line through it. We use CSS to define the {{cssxref("fill")}}, {{cssxref("stroke")}}, and {{cssxref("stroke-width")}} of that path, and add a two-second {{cssxref("transition")}}. We add a {{cssxref(":hover")}} style that contains a slightly different {{cssxref("basic-shape/path", "path()")}}  function; the path has the same number of data points as the default state, making the path animatable.
```

File: files/en-us/web/css/d/index.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- SVG {{SVGAttr("d")}} attribute
+- {{cssxref("fill")}}
+- {{cssxref("stroke")}}
+- {{cssxref("path()")}} function
```
Comment (Josh-Cena) [inline files/en-us/web/css/d/index.md:155]:
```suggestion
- {{cssxref("basic-shape/path", "path()")}}  function
```

**Review by Josh-Cena — APPROVED** [review]
Content LGTM, just fixing a few flaws

**Pull request merged**